troubleshooting/debugging.md

```diff
@@ -191,7 +191,7 @@ If you're running on an old version, it may have been fixed in the latest releas
 
 Or, you may be facing an unresolved or not-yet-discovered issue in a newer version. Downgrading to our latest stable version may work: https://github.com/aws/aws-for-fluent-bit#using-the-stable-tag
 
-In the AWS for Fluent Bit repo, we always attempt to tag bug reports with which versions are affected, for example see [this query](https://github.com/aws/aws-for-fluent-bit/issues?q=is%3Aissue+is%3Aopen+label%3Aaws-2.21.0).  
```

Comment:
I would prefer not to remove this. This was an old version on purpose, since we actually have issues tagged for it, so the user can see it work. I do not believe this sentence implies that a user should use 2.21.0. We can edit the wording here instead to make it more clear- like "for example, see this query which shows that 2.21.0 (very old and out of date) has associated issues".

Reply:
Understood. I'll revert this.